rb/spec/integration/selenium/webdriver/window_spec.rb

```diff
@@ -75,7 +75,7 @@ module WebDriver
         expect(new_pos.y).to eq(target_y)
       end
 
-      it 'gets the rect of the current window', only: {browser: %i[edge edge_chrome firefox ie chrome safari safari_preview]} do
+      it 'gets the rect of the current window', only: {browser: %i[edge firefox ie chrome safari safari_preview]} do
```

Comment:
What browsers wouldn't this work for?? If there is one maybe `except` makes more sense than `only`

Reply:
Good catch, I'll fix